data/101/750/367/101750367.geojson

```diff
@@ -6,8 +6,9 @@
     "date:inception_upper":"0043-12-31",
     "edtf:cessation":"uuuu",
     "edtf:inception":"0043~",
+    "edtf:superseded":"2023-10-25",
```

Comment:
I think the locality record for London should be kept as-is, looks like this record was picked up erroneously because of it's placetype_alt props?

Reply:
Oops, I'll fix.